app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.java

```diff
@@ -36,13 +41,25 @@
 				"courthouse", "embassy", "car_wash", "car_rental",
 				"marketplace", "fuel", "driving_school", "dentist",
 				"doctors", "clinic", "pharmacy", "veterinary",
-				"place_of_worship", "townhall"};
+				"place_of_worship", "townhall", "school", "kindergarten",
+				"shelter", "hospital", "police", "community_centre",
+				"social_facility", "public_building", "university", "college",
+				"theatre", "arts_centre", "bbq", "nursing_home",
+				"ferry_terminal", "childcare", "shower", "prison", "studio",
```

Comment:
surveying prison seems interesting :-) Not.

Reply:
If you tag a prison with a wheelchair key, I would suspect, that this doesn't apply to inmate access but for access for visitors, which should be a valid use-case. Most modernized prisons are accessible for wheelchair users, but this would give a pretty mixed result for Germany.